install_files/securedrop-app-code/DEBIAN/control

```diff
@@ -6,5 +6,5 @@ Homepage: https://securedrop.org
 Package: securedrop-app-code
 Version: 0.8.0~rc1
 Architecture: amd64
-Depends: python-pip,apparmor-utils,gnupg2,haveged,python,python-pip,secure-delete,sqlite,apache2-mpm-worker,libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config
+Depends: python-pip,apparmor-utils,gnupg2,haveged,python,python-pip,secure-delete,sqlite3,apache2-mpm-worker,libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config
```

Comment:
Good news on this front: today I did an upgrade test of this PR in staging (with the Depends field as is here) and the upgrade went fine. This dependency change is ok because the [`sqlite3` package](https://packages.ubuntu.com/trusty/sqlite3) is in `security.ubuntu.com/ubuntu/ trusty-security/main`, so we don't hit #3376:

```
vagrant@app-staging:~$ apt-cache policy sqlite3
sqlite3:
  Installed: 3.8.2-1ubuntu2.1
  Candidate: 3.8.2-1ubuntu2.1
  Version table:
 *** 3.8.2-1ubuntu2.1 0
        500 http://archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages
        500 http://security.ubuntu.com/ubuntu/ trusty-security/main amd64 Packages
        100 /var/lib/dpkg/status
     3.8.2-1ubuntu2 0
        500 http://archive.ubuntu.com/ubuntu/ trusty/main amd64 Packages
```